infra/helper.py

```diff
@@ -961,6 +961,10 @@ def shell(args):
   if args.e:
     env += args.e
 
+  project_language = _get_project_language(args.project_name)
+  if project_language:
+    env.append('FUZZING_LANGUAGE=' + project_language)
```

Comment:
Project language will be always there, so add it in initial env list around line 958

Reply:
I you are sure it is always there :-)

Reply:
Yes we require it now in project.yaml, also in presubmit check.